ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  upi-installer:
+    name: "4.13"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-aro-4.13-nightly
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: ocp-qe-perfscale-ci-tests-aro-cluster-density-v2
+  cron: 0 12 * * 3
+  steps:
+    cluster_profile: azure-qe
+    env:
+      ARO_WORKER_COUNT: "6"
```

Comment:
Is there a reason you picked 6 here?

Reply:
I picked a random number to try scaling. I was thinking we can look into proper profiles after we add support for infra and workload nodes.

Reply:
We could go ahead and see if the infra and workload node set up Liquan added works for aro specific clusters

Reply:
It looks like the cluster is just created using that specified number of workers. I wonder if we'll have to do the same thing we are doing with ROSA with different scaling commands to actually "scale". We'll have to keep an eye out when we get larger number of workers, if installation starts to fail because of quota issue/nodes not coming up in certain times (no action just good to keep watch of)

`az aro create --resource-group ci-***-rg --name ci-***--vnet ci-***-vnet --master-subnet master-subnet --worker-subnet worker-subnet  --pull-secret @"/***/pull-secret"  --worker-vm-size Standard_D8s_v3 --worker-count 6`

Reply:
Spoke to Paige, we will take up infra/workload nodes setup and scaling in a different PR.

Reply:
Here is a jira that I created to track test scenarios coverage for Rosa https://issues.redhat.com/browse/OCPQE-15573 
Same applies to ARO as well.